rd_ui/app/views/users/show.html

```diff
@@ -24,9 +24,21 @@ <h3 class="p-l-5">{{user.name}}</h3>
     </p>
     <div ng-if="currentUser.isAdmin">
       <hr/>
-      <button class="btn btn-default" ng-click="sendPasswordReset()" ng-disabled="disablePasswordResetButton">Send
-        Password Reset Email
-      </button>
+      <div class="form-group">
+        <button class="btn btn-default" ng-click="sendPasswordReset()" ng-disabled="disablePasswordResetButton">Send
+          Password Reset Email
+        </button>
+      </div>
+
+      <div ng-if="passwordResetLink && !disablePasswordResetButton" class="alert alert-success">
+        <p>
+          <strong>The user should receive a link to reset his password by email soon.</strong>
+        </p>
+        <p ng-if="clientConfig.mailSettingsMissing">
+          You can use the following link to reset the password yourself:<br/>
+          {{passwordResetLink}}
+        </p>
```

Comment:
1. Let's show the original message ("should receive by email...") only if email is enabled (`ng-if="!clientConfig.mailSettingsMissing"`).

2. Change the text here to: `You don't have mail server configured, please send the following link to {{user.name}} to reset their password:`.

3. To have the reset link as an `<a>` tag: `<a ng-href="passwordResetLink">{{paswordResetLink}}</a>`. IIRC, it's a full link so the base href doesn't matter.